ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKStreamTest.java

```diff
@@ -262,11 +320,19 @@ public void testSelectWithExpression() {
     final String selectQuery = "SELECT col0, LEN(UCASE(col2)), col3*3+5 FROM test1 WHERE col0 > 100;";
     final PlanNode logicalPlan = planBuilder.buildLogicalPlan(selectQuery);
     final ProjectNode projectNode = (ProjectNode) logicalPlan.getSources().get(0);
-    initialSchemaKStream = new SchemaKStream(logicalPlan.getTheSourceNode().getSchema(), kStream,
-                                             ksqlStream.getKeyField(), new ArrayList<>(), Serdes.String(),
-                                             SchemaKStream.Type.SOURCE, ksqlConfig,
-                                             functionRegistry);
-    final SchemaKStream projectedSchemaKStream = initialSchemaKStream.select(projectNode.getProjectSelectExpressions());
+    initialSchemaKStream = new SchemaKStream(
```

Comment:
Replacing these instances of manually setting up `initialSchemaKStream` with calls to the `buildSchemaKStream()` methods at the bottom of the file would save quite a bit of code.

Reply:
I was being lazy. Will fix.